src/BlockTypes/OrderConfirmation/Totals.php

```diff
@@ -187,7 +187,6 @@ protected function render_order_details_table_item_purchase_note( $order, $produ
 	 * @return string
 	 */
 	protected function render_order_details_table_totals( $order ) {
-		add_filter( 'woocommerce_order_shipping_to_display_shipped_via', '__return_empty_string' );
```

Comment:
Could you explain the context of this change? Why remove this filter, what was it there for originally and why is it no longer needed?

Reply:
Tests were failing because on the new Order Confirmation page, it wasn't displaying the "via xxx" text next to the shipping total. I thought about removing the assert, but that wouldn't be respecting the test. For the sake of fixing tests I prefer to merge this into trunk, and finding out later with the team the reason to remove that span.

Reply:
I went the other way around, removed text was by design requirement. Just waiting for tests to complete again

cc @mikejolley

Reply:
@wavvves can you link to that design requirement or add a pic?

A reminder that we are not skipping PW tests, and we are not taking shortcuts when dealing with failing E2E tests. We should check they don't catch regressions and address the bug fix accordingly. If the test is not reflecting the respected result, tests should be updated.